apps/api/src/schema/study-rooms.ts

```diff
@@ -0,0 +1,34 @@
+import { z } from "@hono/zod-openapi";
+
+export const slotSchema = z.object({
+  studyRoomId: z.string(),
+  start: z.string().openapi({ format: "date-time" }),
+  end: z.string().openapi({ format: "date-time" }),
+  isAvailable: z.boolean(),
+});
+
+export const studyRoomSchema = z.object({
+  id: z.string(),
+  name: z.string(),
+  capacity: z.number().int(),
+  location: z.string(),
+  description: z.string().optional(),
+  directions: z.string().optional(),
+  techEnhanced: z.boolean(),
+  slots: z.array(slotSchema).optional(),
```

Comment:
Is there a reason the slots array is nullable here but not in the GraphQL schema?